tools/taskcluster-proxy/routes.go

```diff
@@ -151,6 +151,7 @@ func (routes *Routes) RootHandler(res http.ResponseWriter, req *http.Request) {
 	defer routes.lock.RUnlock()
 
 	targetPath, err := routes.services.ConvertPath(req.URL)
+	log.Printf("targetPath %s", targetPath)
```

Comment:
do we want to leave this in here or is it debug cruft? /me happy either way

Reply:
It should be removed -- thanks!